### PR TITLE
Reorganized/cleaned up Session classes

### PR DESCRIPTION
**Pull request by niden**
Hello!

* Type: code quality
* Link to issue: #11341 #12833 #13535

**In raising this pull request, I confirm the following (please check boxes):**

- [x] I have read and understood the [Contributing Guidelines](https://github.com/phalcon/cphalcon/blob/master/CONTRIBUTING.md)?
- [x] I have checked that another pull request for this purpose does not exist.
- [ ] I wrote some tests for this PR.

Reorganized the methods and properties in alphabetical order (public, protected, private). Added a new copyright notice at the top of each file

Thanks

### CODE REVIEW

**Comment by niden** [comment]
cc @sergeyklay

**Comment by Jurigag** [comment]
Also remove session service from DI @niden

File: phalcon/session/adapter.zep [hunk 1]
```diff
@@ -1,20 +1,12 @@
...
- |          Eduar Carvajal <eduar@phalconphp.com>                         |
- +------------------------------------------------------------------------+
+ +-------------------------------------------------------------------------+
+ | This file is part of the Phalcon Framework.                             |
+ |                                                                         |
+ | (c) Phalcon Team <team@phalcon.com>                                     |
+ |                                                                         |
+ | For the full copyright and license information, please view the LICENSE |
+ | file that was distributed with this source code.                        |
+ +-------------------------------------------------------------------------+
```
Comment (sergeyklay) [inline phalcon/session/adapter.zep]: Can we use a bit more simple format for this?

```php
/**
 * This file is part of the Phalcon.
 *
 * (c) Phalcon Team <team@phalcon.com>
 *
 * For the full copyright and license information, please view the LICENSE
 * file that was distributed with this source code.
 */
```

File: phalcon/session/adapter.zep [hunk 2]
```diff
@@ -145,23 +134,31 @@ abstract class Adapter implements AdapterInterface
...
 
-		let uniqueId = this->_uniqueId;
-		if !empty uniqueId {
-			let _SESSION[uniqueId . "#" . index] = value;
-			return;
-		}
+	/**
+	 * Get session name
+	 */
+	public function getName() -> string
```
Comment (sergeyklay) [inline phalcon/session/adapter.zep]: Can we use (as usual) tabs instead spaces?

**Review by sergeyklay — CHANGES_REQUESTED** [review]

**Comment by sergeyklay** [comment]
@niden Could you start work on https://github.com/phalcon/cphalcon/issues/12833 ?

**Comment by Jurigag** [comment]
Also add this method from here - https://github.com/phalcon/cphalcon/issues/13535

File: phalcon/session/adapter.zep (hunk 2 above)
Comment (niden) [reply phalcon/session/adapter.zep]: I think it is fixed. I cannot tell from this editor.

**Review by sergeyklay — APPROVED** [review]
LGTM

**Pull request merged**